src/preface.md

```diff
@@ -98,6 +98,17 @@ $ # QEMU
 $ sudo apt-get install qemu-system-arm
 ```
 
+#### Ubuntu 18.04
+- [gcc-arm-none-eabi](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads)
+``` console
+$ tar xvjf gcc-arm-none-eabi-8-2018-q4-major-linux.tar.bz2
+$ mv gcc-arm-none-eabi-<version_downloaded> <your_desired_path> # optional
+$ export PATH=${PATH}:<path_to_arm_none_eabi_folder>/bin # add this line to .bashrc to make persistent
+
```

Comment:
Any reason to prefer an external GDB over `gdb-multiarch`?

I think I'd feel more comfortable describing the use of native packages (if available) and maybe add an extra section about the generic packages (in case of an uncommon distribution or OS).